e2e/rdctl.e2e.spec.ts

```diff
@@ -675,7 +678,7 @@ test.describe('Command server', () => {
           stdout, stderr, error,
         }).toEqual({
           error:  expect.any(Error),
-          stderr: expect.stringContaining(`invalid value for option --container-engine: <"${ myEngine }">; must be 'containerd', 'docker', or 'moby'`),
+          stderr: expect.stringContaining(`Error: invalid value for option --container-engine: <"${ myEngine }">; must be 'containerd', 'moby', or 'docker'`),
```

Comment:
Oh we're not sorting it anymore? Do we need to worry about the order not being consistent?

Reply:
The order comes out of the order as specified in the YAML file. So no more explicit sorting on it.